spring-multirabbit-lib/src/main/java/org/springframework/amqp/rabbit/annotation/MultiRabbitListenerAnnotationBeanPostProcessor.java

```diff
@@ -43,34 +46,22 @@ protected void processAmqpListener(final RabbitListener rabbitListener,
      * Enhance beans with related RabbitAdmin, so as to be filtered when being processed by the RabbitAdmin.
      */
     private void enhanceBeansWithReferenceToRabbitAdmin(final RabbitListener rabbitListener) {
-        final RabbitAdmin rabbitAdmin = getRabbitAdminBean(rabbitListener);
+        final String rabbitAdmin = RabbitAdminNameResolver.resolve(rabbitListener);
```

Comment:
The complete getRabbitAdminBean helper method was removed. Is it's error handling not necessary (anymore)?

Reply:
The error condition was not necessary since `getBeansOfType` returns the object of the expected type and also throws an exception in case the bean does not exist. In addition, no null-check is necessary for the same reason.